Beams/MRML/vtkMRMLRTPlanNode.h

```diff
@@ -187,13 +190,26 @@ class VTK_SLICER_BEAMS_MODULE_MRML_EXPORT vtkMRMLRTPlanNode : public vtkMRMLFold
   /// Set dose engine name (and invoke setting the default beam parameters for the new engine)
   void SetDoseEngineName(const char* engineName);
 
+  // Get optimization engine name
+  vtkGetStringMacro(PlanOptimizerName);
+  /// Set optimization engine name
+  void SetPlanOptimizerName(const char* optimizerName);
+
   /// Get prescription dose
   vtkGetMacro(RxDose, double);
   /// Set prescription dose
   vtkSetMacro(RxDose, double);
 
-  vtkSetVector3Macro(DoseGrid, double);
+  /// Get dose grid spacing

```

Comment:
I thought this was dimensions but now it says spacing. Getting really interesting :)